code-challenges.test.js

```diff
@@ -35,4 +55,28 @@ const votes2 = { upVotes: 2, downVotes: 33 }
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
\ No newline at end of file
+// Pseudo code:
+
+// const tally = (count) => {
+//     const votes = [upVotes,downVotes]
+//     votes.map
+//     return upVotes - downVotes
+// }
+
+
+
+// Function to calculate the end tally based on upVotes and downVotes
+const totalTally = (tally) => {
+// subtract upVotes by downVotes
+    return tally.upVotes - tally.downVotes
+}
+
+
+// Jest test cases
+describe('totalTally', () => {
+    it('return total end tally', () => {
+        expect(totalTally(votes1)).toEqual(11)
+        expect(totalTally(votes2)).toEqual(-31)
+    })
+})
```

Comment:
while you're function and test look good, you need to include the steps of the process in your pseudocode.  This will help me know how I can help you.